oil_lang/expr_to_ast.py

```diff
@@ -140,10 +140,8 @@ def _Trailer(self, base, p_trailer):
       assert p_args.typ == grammar_nt.subscriptlist
       indices = []  # type: List[expr_t]
       n = p_args.NumChildren()
-      for i in xrange(0, n, 2):  # was children[::2]
-        a = p_args.GetChild(i)
-        indices.append(self._Subscript(a))
-      return Subscript(base, indices)
+      a = p_args.GetChild(0)
```

Comment:
If n > 1, we should do something like `p_die("Only 1 subscript is accepted", p_args.GetChild(1).tok)`  which should produce the pretty error highlighting

There is a `test/parse-errors.sh` file that tickles all of these cases